README.md

```diff
@@ -2,11 +2,80 @@
 
 Dissertation project for Sheffield Hallam University MSc in Digital & Technology Solutions
 
-#### Overview
+## Overview
 
 The purpose of this application is to compare the read and write performances of 
 a Kafka Streams backed State store against a MongoDB based application.
```

Comment:
I'd mention here that you are using spring boot as that could heavily influence your results.

Make it a test between Spring Kafka Streams, & Spring MongoDB